docs/reference/4-command-line-tools/1-goliothctl/goliothctl_lightdb_listen.md

```diff
@@ -0,0 +1,51 @@
+---
+id: goliothctl_lightdb_listen
+title: "goliothctl lightdb listen"
+slug: goliothctl_lightdb_listen
+sidebar_label: goliothctl lightdb listen
+url: /reference/command-line-tools/goliothctl/goliothctl_lightdb_listen/
+hide_title: true
+---
+## goliothctl lightdb listen
+
+Listen for device state changes on the LightDB root path or a specific path.
+Use the device name, device id or hardware id of the device to select the device.
+
+```bash
+goliothctl lightdb listen [device name] <path>
+goliothctl lightdb listen [flags] <path>
+```
+
+### Examples
+
+```bash
+# Listen to all lightdb messages from a device
+> goliothctl lightdb listen <device name>
+> goliothctl lightdb listen --id <device id>
+> goliothctl lightdb listen --hwid <hardware id>
+
+# Listen to a specific path for lightdb messages from a device
+> goliothctl lightdb listen <device name> <path>
+> goliothctl lightdb listen --id <device id> <path>
+> goliothctl lightdb listen --hwid <hardware id> <path>
+```
+
+### Options
+
+```bash
+  -h, --help               help for lightdb listen
```

Comment:
Please confirm that I have used the correct help text here.

Reply:
I don't think it's a problem Mike.